src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/letters/InvalidCharacterCheckTest.java

```diff
@@ -56,4 +61,22 @@ public void checkInvalidCharactersInNames() {
         assertEquals(String.format(NEW_LINE_ERROR, "Respondent Double  Space and New\nLine",
                 casedata.getEthosCaseReference(), "letter"), errors.get(3));
     }
+
+    @Test
+    public void checkInvalidCharsTestListingTypes() {
+        ListingDetails listingDetails = new ListingDetails();
+        listingDetails.setCaseData(new ListingData());
+        ListingTypeItem item = new ListingTypeItem();
+        item.setId(UUID.randomUUID().toString());
+        ListingType value = new ListingType();
+        value.setRespondent("Forename" + "\n" + "Surname");
```

Comment:
"Forename\nSurname" ? same for the assertion below

Reply:
done